pkgs/development/web/nhost/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "nhost";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "nhost";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = "sha256-mjhNZwbYc5RKxDBMmXCllKhHNJYwau5rQABUjKYIkf0=";
+  };
+
+  vendorSha256 = null;
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/nhost
+  '';
+
+  meta = with lib; {
+    description = "The Nhost CLI is used to get a local development environment.";
```

Comment:
This should not end with a period.

See: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

Reply:
done